packages/theme/src/components/prose.ts

```diff
@@ -0,0 +1,173 @@
+import { mode, SystemStyleFunction } from "@chakra-ui/theme-tools"
+
+import { getComponentBaseStyle } from ".."
+
+const baseStyle: SystemStyleFunction = (props) => {
+  const { theme } = props
+  const headingBase = getComponentBaseStyle(theme, "Heading")
+  const linkBase = getComponentBaseStyle(theme, "Link")
```

Comment:
I think Markdown headings and links could be styled differently from the components. Feel free to create their own standalone styles. This code assumes that the user hasn't deleted, omitted or changed the styles for Heading and/or Link.